libraries/botbuilder/etc/botbuilder.api.md

```diff
@@ -458,7 +458,7 @@ export class TeamsInfo {
     static getTeamMember(context: TurnContext, teamId?: string, userId?: string): Promise<TeamsChannelAccount>;
     // @deprecated
     static getTeamMembers(context: TurnContext, teamId?: string): Promise<TeamsChannelAccount[]>;
-    static sendMeetingNotification(context: TurnContext, notification: TeamsMeetingNotification, meetingId?: string): Promise<TeamsMeetingNotificationRecipientFailureInfos | {}>;
+    static sendMeetingNotification(context: TurnContext, notification: MeetingNotification, meetingId?: string): Promise<MeetingNotificationRecipientFailureInfos | {}>;
```

Comment:
Similar to my other comment - can we simply do MeetingNotificationRecipientFailureInfo[]?

Reply:
And is there a reason why we use `MeetingNotificationRecipientFalureInfos | {}` over MeetingNotificationResponse? ex: What I am expecting is similar to `Promise<TeamsPagedMembersResult>`

Reply:
Agreed

Reply:
Updated